### DISCUSSION TITLE
Add a shorcut to switch between interactive window and editor

### DISCUSSION THREAD

**Discussion by stevenlis** [microsoft/vscode-jupyter#11230] [New Ideas]
Is there a way to add a shortcut to switch the focus between interactive window (code entering area) and editor? 
I couldn't find a way to do so.
![CleanShot 2022-08-30 at 17 25 11](https://user-images.githubusercontent.com/33796896/187546238-0cb0b46f-4e86-4396-bdcb-e911ae3a636e.png)

**Comment by IanMatthewHuff** [comment]
@StevenLi-DS When I'm working like this I usually use the VS Code editor group switching command Cmd (Ctrl on Windows) + 1, 2, 3 to swap between editor groups. When the Interactive Window editor group gets focus it should set the cursor to the input box.

**Reply by stevenlis** [reply]
@IanMatthewHuff oh.. thanks man. I didn't realize it's cmd on Mac and tried with crtl + 1 2 3 but it only switched between my tabs. Now I realized that you can use ctrl to switch tabs and cmd to switch panels.

**Reply by stevenlis** [reply]
@IanMatthewHuff I am trying to use the new `interactive.input.focus` but I'm stuck in the interactive window and didn't find a input to switch back to editor focus. Am i missing something?

**Comment by stevenlis** [comment]
@IanMatthewHuff would you happen to know if there is anyway to resize the panels with keyboard shortcuts? 

https://user-images.githubusercontent.com/33796896/188014636-17168005-5213-4654-8b99-efc38eaa05f1.mp4

**Reply by IanMatthewHuff** [reply]
Hmm, poking around the best that I'm seeing is the 'Increase / Decrease Current View Size" and "Increase / Decrease Editor Width" commands. They are currently unbound, so you would have to add your own keyboard shortcut. And they don't really allow full precise resizing. Best that I can find though.

**Reply by stevenlis** [reply]
Thanks man, they do seem to be what I need!
